auacm/app/modules/submission_manager/judge.py

```diff
@@ -65,9 +62,6 @@ def evaluate(submission, uploaded_file, time_limit=10):
     :param uploaded_file: the uploaded file
     :return: the status of the submission (one of the status constants above)
     """
-    directory = directory_for_submission(submission)
-    os.mkdir(directory)
-    uploaded_file.save(os.path.join(directory, uploaded_file.filename))
```

Comment:
Removing these breaks the judge tests. I know it's necessary since you can't save the files in a separate thread, but we need to figure that out.